src/libs/Clipboard/index.js

```diff
@@ -0,0 +1,52 @@
+// on Web/desktop this import will be replaced with `react-native-web` so we have to import the named export;
+import {Clipboard as base} from '@react-native-community/clipboard';
+
+
+class Clipboard extends base {
+    /**
+     * Enable copying the images on Web/Desktop
+     *
+     * @static
+     * @param {String} html
+     * @returns {Boolean}
+     * @memberof Clipboard
+     */
+    static setImage(html) {
```

Comment:
Hmmm I thought in that in the [linked issue](https://github.com/Expensify/Expensify.cash/issues/1778#issuecomment-804297647) we discussed that we weren't going to try to implement image-copying just yet.

I know you can probably make it work on web right now, but per the [Expensify.cash philosophy](https://github.com/Expensify/Expensify.cash#philosophy), we want _all_ features to work on _all_ platforms, and until we're ready to make this work on _all_ platforms, we don't want to implement this for just one platform. That way, we're staying committed to never letting one platform fall behind, and our codebase is truly cross-platform. Does that make sense?

Sorry you've already put some work into this, but maybe we'll be able to back-reference this one day.

Reply:
>That way, we're staying committed to never letting one platform fall behind, and our codebase is truly cross-platform. 

I agree with this one. So I am taking out this code and I will just export the Clipboard package here. We do need to separate the native and web as the library exports are different.